src/Appwrite/Platform/Modules/DevKeys/Http/DevKeys/CreateKey.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Appwrite\Platform\Modules\DevKeys\Http\DevKeys;
+
+use Appwrite\Extend\Exception;
+use Appwrite\Utopia\Response;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Permission;
+use Utopia\Database\Helpers\Role;
+use Utopia\Database\Validator\Datetime as DatetimeValidator;
+use Utopia\Database\Validator\UID;
+use Utopia\Platform\Action;
+use Utopia\Platform\Scope\HTTP;
+use Utopia\Validator\Text;
+
+class CreateKey extends Action
+{
+    use HTTP;
+    public static function getName()
+    {
+        return 'createKey';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->setHttpMethod(Action::HTTP_REQUEST_METHOD_POST)
+            ->setHttpPath('/v1/projects/:projectId/development-keys')
+            ->desc('Create key')
+            ->groups(['api', 'projects'])
+            ->label('scope', 'projects.write')
+            ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+            ->label('sdk.namespace', 'projects')
+            ->label('sdk.method', 'createKey')
```

Comment:
createDevKey, to be consistent with the rest.

Reply:
updated.